### PR TITLE
Force Reirect for SL packages delivered via S3 

### PR DESCRIPTION
**Pull request by cklosowski**
#18

### CODE REVIEW

File: edd_s3.php [hunk 1]
```diff
@@ -621,6 +633,29 @@ public function add_set_download_method( $download, $email, $payment, $args = ar
...
+	 * Compability for setting the redirect method with the Software Licensing toknized URLs
+	 *
+	 * @since  2.3
+	 * @param  int $id         The Download ID
+	 * @param  string $hash    The Hash passed to SL
+	 * @param  string $license The License ID requesting the update package
+	 * @param  int    $expires The expiration of the link
+	 * @return void
+	 */
+	public function all_sl_set_download_method( $id, $hash, $license, $expires ) {
```
Comment (pippinsplugins) [inline edd_s3.php]: What's the `all` stand for here?

**Comment by pippinsplugins** [comment]
Aside from my one question, this looks good.

**Comment by pippinsplugins** [comment]
@cklosowski Looks like this got missed.

File: edd_s3.php [hunk 2]
```diff
@@ -769,6 +804,11 @@ public function send_fes_files_to_s3( $files = array(), $post_id = 0 ) {
 	}
 }
 
+/**
+ * Load the S3 plugin for EDD
+ *
+ * @return void
```
Comment (thefrosty) [inline edd_s3.php]: 👀 This doesn't look like it returns void to me.

**Comment by cklosowski** [comment]
Closing this PR for a new one, based off recent `master`

**Pull request closed without merging**